### PR TITLE
Mathieu/feat/routes

### PR DESCRIPTION
**Pull request by MathieuRobilliard**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "remove",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/react-router": "^4.0.15",
+    "@types/react-router-dom": "^4.0.8",
```
Comment (Porquepix) [inline package.json]: @types must be in dev-dependencies.

File: src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
+import App from './App';
+import LoginPage from './LoginPage';
+
+import registerServiceWorker from './registerServiceWorker';
+import './index.css';
+
+/*
+ReactDOM.render(
```
Comment (Porquepix) [inline src/index.tsx]: You can delete these comments.

File: src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <BrowserRouter>
+  <Switch>
+      <Route path="/App" component={ App }/>
+      <Route path="/Login" component={ LoginPage }/>
+  </Switch>
+  </BrowserRouter>, document.getElementById('root')
+);
+registerServiceWorker();
+/*
+ReactDOM.render(
```
Comment (Porquepix) [inline src/index.tsx]: These comments too can be deleted.

File: src/index.tsx (hunk 2 above)
Comment (MathieuRobilliard) [reply src/index.tsx]: Done in the branch Mathieu/feat/LoginPage

File: src/index.tsx (hunk 3 above)
Comment (MathieuRobilliard) [reply src/index.tsx]: Done in the branch Mathieu/feat/LoginPage

File: package.json (hunk 1 above)
Comment (MathieuRobilliard) [reply package.json]: Done in the branch Mathieu/feat/LoginPage

**Pull request merged**